run_local.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+sudo cp -rp executable/* .
```

Comment:
do we need a sudo to copy files?

Reply:
Dont quite sure, but left it for `-p` option, which resposible for saving the righs of files (such as chmod and so on)